src/epub/text/the-thing-on-the-doorstep.xhtml

```diff
@@ -14,15 +14,15 @@
 				<hr/>
 				<p>Derby went through Miskatonic University in Arkham, since his parents would not let him board away from them. He entered at sixteen and completed his course in three years, majoring in English and French literature and receiving high marks in everything but mathematics and the sciences. He mingled very little with the other students, though looking enviously at the “daring” or “Bohemian” set⁠—whose superficially “smart” language and meaninglessly ironic pose he aped, and whose dubious conduct he wished he dared adopt.</p>
 				<p>What he did do was to become an almost fanatical devotee of subterranean magical lore, for which Miskatonic’s library was and is famous. Always a dweller on the surface of fantasy and strangeness, he now delved deep into the actual runes and riddles left by a fabulous past for the guidance or puzzlement of posterity. He read things like the frightful <i>Book of Eibon</i>, the <i>Unaussprechlichen Kulten</i> of von Junzt, and the forbidden <i>Necronomicon</i> of the mad Arab Abdul Alhazred, though he did not tell his parents he had seen them. Edward was twenty when my son and only child was born, and seemed pleased when I named the newcomer Edward Derby Upton, after him.</p>
-				<p>By the time he was twenty-five Edward Derby was a prodigiously learned man and a fairly well-known poet and <em>fantaisiste</em>, though his lack of contacts and responsibilities had slowed down his literary growth by making his products derivative and over-bookish. I was perhaps his closest friend⁠—finding him an inexhaustible mine of vital theoretical topics, while he relied on me for advice in whatever matters he did not wish to refer to his parents. He remained single⁠—more through shyness, inertia and parental protectiveness than through inclination⁠—and moved in society only to the slightest and most perfunctory extent. When the war came both health and ingrained timidity kept him at home. I went to Plattsburg for a commission, but never got overseas.</p>
+				<p>By the time he was twenty-five Edward Derby was a prodigiously learned man and a fairly well-known poet and <em>fantasist</em>, though his lack of contacts and responsibilities had slowed down his literary growth by making his products derivative and over-bookish. I was perhaps his closest friend⁠—finding him an inexhaustible mine of vital theoretical topics, while he relied on me for advice in whatever matters he did not wish to refer to his parents. He remained single⁠—more through shyness, inertia and parental protectiveness than through inclination⁠—and moved in society only to the slightest and most perfunctory extent. When the war came both health and ingrained timidity kept him at home. I went to Plattsburgh for a commission, but never got overseas.</p>
```

Comment:
This is incorrect, `fantaisiste` is a French word, not English emphasis. This should be reverted and changed to `<i xml:lang="fr">`